tasks/opensuse/install.yml

```
@@ -0,0 +1,21 @@
+---
+- name: OpenSUE | Install Repo
```

To make the output more readable:

```suggestion
- name: OpenSUE | Install Repo - Tumbleweed
```